proto/v1/event.proto

```diff
@@ -20,8 +20,8 @@ package datacommons.v1;
 message EventCollection {
   message GeoLocation {
     message Point {
-      double latitude = 1;
-      double longitude = 2;
+      optional double latitude = 1;
```

Comment:
please add a note here so we don't drop it in the future

Reply:
done in a followup PR: https://github.com/datacommonsorg/mixer/pull/1025